<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>580</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>981</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajout une reparation</string>
     </attribute>
     <widget class="QGroupBox" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>431</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_reparation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Piece</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Facture</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ID_REPARATION">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_piece">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_facture">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>250</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_debut">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Panne</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date_dispo</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date_debut</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DATE_DISPO">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>120</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QComboBox" name="le_panne">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>panne de batterie</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>fuite du joint</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>refroidissement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>roues</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>huile moteur</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher une reparation</string>
     </attribute>
     <widget class="QTableView" name="tab_reparation">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>11</y>
        <width>501</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="date_dispo_modifier">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_dispo">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>120</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date_dispo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_reparation">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>80</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Id_reparation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_button">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>240</y>
        <width>191</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Trier par Facture</string>
      </property>
      <widget class="QRadioButton" name="radioButton_croissant">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri Croissant</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_dcroissant">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri decroissant</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer une reparation</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_supp">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id_reparation</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Statistique</string>
     </attribute>
     <widget class="QPushButton" name="statsmed">
      <property name="geometry">
       <rect>
        <x>682</x>
        <y>320</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Appliquer Statistique</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Envoyer Gmail</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1281</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>Envoie email</string>
      </property>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>551</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>liste</string>
       </property>
       <widget class="QTableView" name="tableView_email">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>531</width>
          <height>131</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="lineEdit_from_email">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>331</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string>from</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_psswrd_email">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>180</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string>password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_to_email">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>591</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string>to</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_atchmnt_email">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>491</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string>attachment</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_subject_email">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>491</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string>subject</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_email">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>571</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_browse_email">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>240</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_send_email">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>270</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
